### PR TITLE
Add initial Pager Duty platform docs

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

-  Add initial Pager Duty platform docs

## Testing

Preview: https://add-pager-duty.botkube-docs.pages.dev/next/

## Related issue(s)

- https://github.com/kubeshop/botkube-cloud/issues/835

### CODE REVIEW

File: docs/installation/pagerduty/index.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+To see the alerts in the PagerDuty web app, click **Incidents** tab and select **Alerts**:
+
+![pager_duty_app_alert.png](assets/pager_duty_app_alert.png)
+
+### Changes
+
+A [change](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTgy-send-a-change-event) indicates a non-problematic update within a system. Botkube forwards the following types of change events to PagerDuty:
+
+- [Kubernetes'](../../configuration/source/kubernetes.md) `create` and `update` events.
+- The [ArgoCD](../../configuration/source/argocd.md) application events including creation, updates, deployment, and deletion. Additionally, the synchronization statuses include running, unknown, and succeeded events.
```
Comment (pkosiec) [inline docs/installation/pagerduty/index.md:102]: WDYT about something like that:
```suggestion
- - **[Kubernetes](../../configuration/source/kubernetes.md) plugin:** `create` and `update` events.
- **[ArgoCD](../../configuration/source/argocd.md) plugin:** application events including creation, updates, deployment, and deletion. Additionally, the synchronization statuses include running, unknown, and succeeded events.
```

If you like it, please also apply it in the "Alerts" section 👍

**Review by pkosiec — APPROVED** [review]
LGTM but please fix the PagerDuty spelling everywhere before the merge. Also, please take a look at the other minor comments 🙂

File: docs/installation/index.mdx [hunk 2]
```diff
@@ -48,6 +48,7 @@ Compare our unidirectional integrations:
     { name: "Discord", image: "/images/integrations/discord.svg", link: "installation/discord" },
     { name: "Teams", image: "/images/integrations/msteams.svg", link: "installation/teams" },
     { name: "Elasticsearch", image: "/images/integrations/elasticsearch.svg", link: "installation/elasticsearch" },
+    { name: "Pager Duty", image: "/images/integrations/pager-duty.svg", link: "installation/pagerduty" },
     { name: "Webhook", image: "/images/integrations/webhooks.svg", link: "installation/webhook" },
   ]}
 />
```
[the remark this answers is not in the document]
Comment (mszostok) [reply docs/installation/index.mdx:54]: I think the main idea is not that bad, as it's a good feature to enable multiple platforms with a single agent. I just made it simpler so we don't need to maintain the list in yet another place 👍

File: docs/installation/pagerduty/index.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+For a detailed guide, navigate to PagerDuty's support page [Services and Integrations](https://support.pagerduty.com/docs/services-and-integrations#create-a-service).
+:::
+
+To create a service in the Pager Duty web app:
+
+1. Go to **Services** -> **Service Directory** and click **New Service**. On the next screen you will be guided through several steps.
+   1. Enter **Name**. We suggest to put a cluster name, for example: "Production EU".
+   2. Click **Next** to continue.
+2. Select Escalation Policy that suits you best. Click **Next** to continue.
+3. Select the **Events API V2** integration.
```
[the remark this answers is not in the document]
Comment (mszostok) [reply docs/installation/pagerduty/index.md:51]: covered 👍 I also added a screenshot for the last section to make sure user knows where to find the integration key.

**Pull request merged**